velox/functions/prestosql/tests/ZipTest.cpp

```diff
@@ -80,7 +46,8 @@ TEST_F(ZipTest, simpleInt) {
   auto rowVector = makeRowVector({firstResult, secondResult});
 
   // create the expected ArrayVector
-  auto expected = createArrayVector({0, 4, 7}, {4, 3, 3}, rowVector);
+  auto expected =
+      vectorMaker_.createArrayVector({0, 4, 7}, {4, 3, 3}, rowVector);
```

Comment:
A common pattern is to add xxx method to VectorMaker and createXxx method to FunctionTestBase. It would be nice to follow it here as well.